com.rlabrecque.steamworks.net/Runtime/com.rlabrecque.steamworks.net.asmdef

```diff
@@ -14,7 +14,9 @@
     "overrideReferences": true,
     "precompiledReferences": [],
     "autoReferenced": true,
-    "defineConstraints": [],
+    "defineConstraints": [
+        "UNITY_EDITOR || STEAMWORKS_NET"
+    ],
```

Comment:
Hey Dylan, one thing I am slightly concerned about is what this implies out of the box for non-editor packages. Our overall enabling/disabling situation in general probably needs a complete revisit, and I've honestly kind of lost track of say whether one has to pre-define STEAMWORKS_NET somewhere, or if it will get automatically defined [unless you disable it somehow?], etc. Any thoughts/insights? I won't really have time to look into this any time soon.

Aside from this piece the rest looks great.